beacon-chain/p2p/discovery.go

```diff
@@ -98,25 +97,31 @@ func (s *Service) createListener(
 	ipAddr net.IP,
 	privKey *ecdsa.PrivateKey,
 ) (*discover.UDPv5, error) {
-	// assume ip is either ipv4 or ipv6
+	// Listen to all network interfaces
+	// for both ip protocols.
 	networkVersion := ""
```

Comment:
```suggestion
	var networkVersion string
```
Just for consistency with `createListener()` code + it kind of manifests that value will be set somewhere down the road (with assignment it is more like empty string "" is a valid value).